rest/taskrouter/workers/instance/get/example-1/example-1.5.x.cs

```diff
@@ -0,0 +1,29 @@
+// Download the twilio-csharp library from
+// https://www.twilio.com/docs/libraries/csharp#installation
+using Newtonsoft.Json.Linq;
+using System;
+using System.Net;
+using Twilio;
+using Twilio.Rest.Taskrouter.V1.Workspace;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        const string workspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string WorkerSid = "WKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+        TwilioClient.Init(accountSid, authToken);
+
+        var worker = WorkerResource.Fetch(workspaceSid, WorkerSid);
+        Console.WriteLine(worker.Attributes);
+
+        // alternatively
+        var attributes = JObject.Parse(worker.Attributes);
+        string value = attributes["foo"].Value<string>();
```

Comment:
There is no real benefit on extracting the `value` variable, furthermore if that's the case we don't need to use a concrete type, we can rely on `var` and let the compiler to infer the type for us. 😉

I would write this part of the example as follows:

```csharp
Console.WriteLIne(attributes["foo"]);
```